src/components/touchable-opacity-or-view/TouchableOpacityOrView.tsx

```diff
@@ -5,13 +5,22 @@ const TouchableOpacityOrView = ({
   style,
   onClick,
   children,
+  accessibilityHint,
+  accessibilityLabel,
 }: {
   style: ViewStyle;
   children: ReactNode;
   onClick?: () => void;
+  accessibilityHint?: string;
+  accessibilityLabel?: string;
```

Comment:
A `View` can have also have accessibility fields. Might be better to just add the `AccessibilityProps`-type here, and if provided add to both `View` and `TouchableOpacity`?